node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -27,14 +27,18 @@ object Main {
 
   def main(args: Array[String]): Unit = {
 
+    implicit val scheduler: Scheduler = Scheduler.computation(
```

Comment:
Why `computation`? It uses `ForkJoinPool` internally, is `mainProgram` of this _fork-join_ characteristic?